### DISCUSSION TITLE
[Suggestion] Default action when clicking on JSON property object

### DISCUSSION THREAD

**Discussion by ryanbuening** [datalust/seq-tickets#1141] [Ideas]
I think it would be nice to make the default action when clicking on a JSON object property to either expand or collapse.

You would still have all the options if the user clicked on the circle/idea icon to the left.

![image](https://user-images.githubusercontent.com/812974/108540799-a2235580-72af-11eb-8157-d7ca27fa857a.png)

**Comment by nblumhardt** [comment]
Thanks for the suggestion! We'll give it some thought :+1:
